pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -43,6 +43,7 @@ import (
 	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
 	"kubevirt.io/kubevirt/pkg/libvmi"
 	"kubevirt.io/kubevirt/pkg/pointer"
+	kvpointer "kubevirt.io/kubevirt/pkg/pointer"
```

Comment:
We don't need to import this again, guess this came in with the rebase?

./me makes a note to enable linting in `virt-api`.

Reply:
this was added automatically by running `make`. I'll remove it.

Reply:
Fixed it.